app/controllers/bus_stops_controller.rb

```diff
@@ -40,6 +59,20 @@ def name_search
     end
   end
 
+  def outdated
+    @date = Date.parse(params[:date]) rescue 1.month.ago.to_date
```

Comment:
Why is the rescued value `1.month.ago.to_date`?

Reply:
It needs to be something. I'm open to suggestions.